app/templates/gentelella/users/events/index.html

```diff
@@ -91,7 +91,15 @@
     <script src="{{ url_for('static', filename='vendor/datatables.net-bs/js/dataTables.bootstrap.min.js') }}"></script>
     <script src="{{ url_for('static', filename='vendor/bootstrap-progressbar/bootstrap-progressbar.min.js') }}"></script>
     <script type="text/javascript">
-        $('.table').DataTable();
+    $('.table').DataTable( {"columnDefs": [
```

Comment:
The alignment is still not proper here. Always ensure your code is readable. Something like this 

```
$('.table').DataTable({
    "columnDefs" : [
      {"width" : "8%", "targets" : 0},
      {"width" : "15%", "targets" : 1},
      {"width" : "23%", "targets" : 2},
      {"width" : "10%", "targets" : 3},
      {"width" : "10%", "targets" : 4},
      {"width" : "10%", "targets" : 6}
    ]
  });
```